pkg/daemon/rafs.go

```diff
@@ -95,18 +94,18 @@ func (rs *rafsSet) List() map[string]*Rafs {
 	return instances
 }
 
-func (rs *rafsSet) ListUnlocked() map[string]*Rafs {
+func (rs *rafsSet) ListLocked() map[string]*Rafs {
```

Comment:
`Locked` suffix indicates this function has lock inside its body?

Reply:
The `Locked` suffix means this function should be called with related lock held. It's a code style from Linux kernel:)